src/holodeck/util.py

```diff
@@ -158,3 +158,18 @@ def draw_point(env, loc, color=None, thickness=10.0):
     color = [255, 0, 0] if color is None else color
     command_to_send = DebugDrawCommand(3, loc, [0, 0, 0], color, thickness)
     env._enqueue_command(command_to_send)
+
+
+def logs_path():
+    """Gets path for logs.
+
+    Returns:
+        :obj:`str`: The file path of where the logs are located
+    """
+
+    if os.name == "nt":
+        print(os.path.join(get_holodeck_path(), r'\worlds\DefaultWorlds\WindowsNoEditor\Holodeck\Saved\Logs'))
+        return os.path.join(get_holodeck_path(), r'\worlds\DefaultWorlds\WindowsNoEditor\Holodeck\Saved\Logs')
+    if os.name == "posix":
+        print(os.path.join(get_holodeck_path(), r'/worlds/DefaultWorlds/LinuxNoEditor/Holodeck/Saved/Logs/'))
+        return os.path.join(get_holodeck_path(), r'/worlds/DefaultWorlds/LinuxNoEditor/Holodeck/Saved/Logs/')
```

Comment:
Seems like there's some duplication here, and this doesn't account for all worlds. Could we do something like this?
```python
return [
    os.path.join(
        get_holodeck_path(),
        "worlds",
        package,
        "{}NoEditor".format(get_os_key()),
        "Holodeck",
        "Saved",
        "Logs",
    )
    for package in holodeck.packagemanager.installed_packages()
]

```

`os.path.join` will join as many paths as you want, which you can use to avoid having to specify the entire path twice with Windows- and Unix-style slashes. There might be an even better way to do this--I haven't checked.

Reply:
I tried this:
```python
    paths = []

    for package in holodeck.packagemanager.installed_packages():
        paths.append(
            os.path.join(
                get_holodeck_path(),
                'worlds',
                package,
                '{}NoEditor'.format(get_os_key()),
                "Holodeck",
                "Saved",
                "Logs",
            )
        )

    return paths
```
it returns: 
```
['C:\\Users\\Chris\\AppData\\Local\\holodeck\\0.3.2.dev0\\worlds\\DefaultWorlds\\WindowsNoEditor\\Holodeck\\Saved\\Logs']
```
it also brings this up when I call the function:
```
**********************************************
* You have old versions of Holodeck packages *
**********************************************
Use packagemanager.prune() to delete old packages
Versions: ['0.3.2.dev0 (old)']
Place an `ignore_old_packages` file in C:\Users\Chris\AppData\Local\holodeck to suppress this message
```
it that's alright, then I'll keep it like this